### PR TITLE
Re-adds borers.

### PR DESCRIPTION
**Pull request by MacHac**
Originally by Oisin100.
Sorry this took so long, I got lost in some git shenanigans.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+
+	var content = ""
+	content += "<p>Chemicals: <span id='chemicals'>[chemicals]</span></p>"
+	content += "<p>Influence: <span id='influence'>[influence]</span>%</p>"
+
+	content += "<table>"
+
+	for(var/datum in typesof(/datum/borer_chem))
+		var/datum/borer_chem/C = new datum()
+		if(C.chemname && C.needed_influence < influence)
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm:73]: Could you change `<` to `<=` while yer at it? Kthx.

File: code/modules/mob/living/simple_animal/borer/borer_topic.dm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		if(!istype(C, /datum/borer_chem))
+			return
+
+		if(!C || !victim || controlling || !src || stat)
+			return
+
+		if(!istype(C, /datum/borer_chem))
+			return
+
+		if(chemicals < C.chemuse)
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_topic.dm:26]: Could probaably do with a influence check here somewhere.

**Comment by ShadowDeath6** [comment]
My borer PR recently got merged, might want to give that a look, since it got rid of influence.

**Comment by ShadowDeath6** [comment]
I madea borer PR that recently got merged, might want to give that a look since it got rid of influence and some other small changes.

**Pull request closed without merging**